### PR TITLE
feat: add school page

### PR DESCRIPTION
**Pull request by Kojo-Brown**
First draft for schools page

### CODE REVIEW

File: src/app/dashboard/schools/schools.component.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    @if (isLoadingResults) {
+      <div class="empty-table loading-results">
+        @if (isLoadingResults) {
+          <mat-spinner></mat-spinner>
+        }
+      </div>
+    }
+    @if (!isLoadingResults && !data.length) {
+      <div class="empty-table no-results">
+        <span>No users found</span>
```
Comment (ghost) [inline src/app/dashboard/schools/schools.component.html]: Schools

Comment (ghost) [inline src/app/dashboard/schools/schools.component.html]: Schools

File: src/app/dashboard/schools/schools.component.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      <div class="actions__left">
+        <span>Schools</span>
+        <div class="search__input__wrapper">
+          <app-rounded-input
+            [control]="searchValue"
+            placeholder="Search by name" />
+        </div>
+      </div>
+      <div class="actions__right">
+        <!-- <button
```
Comment (ghost) [inline src/app/dashboard/schools/schools.component.html]: You still need this?

**Review by ghost — DISMISSED** [review]
The class and css references user

File: src/app/dashboard/schools/schools.component.html (hunk 2 above)
Comment (Kojo-Brown) [reply src/app/dashboard/schools/schools.component.html]: Maybe not. What could we possibly filter by?

**Review by ghost — APPROVED** [review]

**Pull request merged**